### PR TITLE
Update deps, format with Prettier

### PR DESCRIPTION
**Pull request by drwpow**
Was working on a side-project and some of @pika/pack’s deps are a little out-of-date.

I re-built my side project using the updated version, and the code seemed to be identical. All tests passed, no major changes. 👍🏻 

Prettier was also run over the project, but otherwise code wasn’t changed.

Should help get rid of some security warnings with `@pika/pack`!

### CODE REVIEW

File: FUNDING.yml [hunk 1]
```diff
@@ -1 +0,0 @@
-patreon: pikapkg
```
Comment (drwpow) [inline FUNDING.yml:1]: Removes funding

File: package.json [hunk 2]
```diff
@@ -40,30 +40,29 @@
...
-    "invariant": "^2.2.0",
+    "invariant": "^2.2.4",
     "is-builtin-module": "^3.0.0",
     "is-ci": "^2.0.0",
-    "loud-rejection": "^2.1.0",
-    "mkdirp": "^0.5.1",
-    "np": "^5.0.2",
-    "rimraf": "^2.5.0",
-    "strip-ansi": "^5.2.0",
+    "kleur": "^4.1.3",
```
Comment (drwpow) [inline package.json:50]: Chalk -> kleur b/c why not? 🤷🏻

File: package.json [hunk 3]
```diff
@@ -40,30 +40,29 @@
...
+    "loud-rejection": "^2.2.0",
+    "mkdirp": "^1.0.4",
+    "np": "^7.1.0",
```
Comment (drwpow) [inline package.json:53]: `np` seemed to be the biggest offender of out-of-date deps. This fixes that.

File: src/commands/build.ts [hunk 4]
```diff
@@ -48,9 +48,9 @@ export class Build {
       out,
       cwd,
       reporter: {
-        info: msg => reporter.log(chalk.dim(`      » ${msg}`)),
-        warning: msg => reporter.log(chalk.yellow(`      » ${msg}`)),
-        success: msg => reporter.log(chalk.green(`      » ${msg}`)),
+        info: (msg) => reporter.log(chalk.dim(`      » ${msg}`)),
```
Comment (drwpow) [inline src/commands/build.ts:51]: No code changes here and elsewhere; only Prettier formatting

**Comment by FredKSchott** [comment]
Oh wow this got really out of date. Thanks for bumping!

**Pull request merged**